external/api/src/main/java/org/codice/compliance/saml/plugin/IdpPostResponse.java

```diff
@@ -51,21 +52,25 @@ public IdpPostResponse(Response response) {
 
     if (responseForm != null) {
       parseAndSetFormValues();
+    } else {
+      // Purely for debugging purposes
+      responseBodyString = response.then().extract().body().asString();
```

Comment:
✏️ While this isn't a super-expensive operation, it might be worth doing lazy initialization of it (harder in Java, I know) if it's ever invoked. Maybe an AtomicReference with a method to fill it? Not sure.

Reply:
Well I'd have to save the RestAssured Response object (named `response` here) to a field. But that would work if I just save it to a field and then extract the body the one time I need to debug it. I'll give that a try.